docs/user/filesystem/architecture.rst

```diff
@@ -0,0 +1,70 @@
+FileSystem Datastore Architecture
+=================================
+
+The GeoMesa FileSystem Datastore (GeoMesa FSDS) takes advantage of the performance characteristics of modern
+cloud-native and distributed filesystems to scale bulk analytic queries. The FSDS is a good choice for doing bulk egress
+queries or large analytic jobs using frameworks such as Spark SQL and MapReduce. The FSDS differs from other queries
+in that ingest latency is traded for high-throughput query. The FSDS pairs well with low-latency ingest systems such as
+HBase or Kafka to provide an optimal pairing of "hot", "warm" storage options.
+
+The GeoMesa FSDS consists of a few primary components:
+
+* **FileSystem** - A separately managed storage system that implements the GeoMesa FileSystem API
+* **Partition Scheme** - A stategy for laying out data on within the filesystem
+* **Storage Format** - A defined format or encoding to store data in files
+* **Query Engine** - A query engine or client to fulfill queries and run analytic jobs
+
+
+FileSystem
+----------
+
+GeoMesa FSDS can utilize any filesystem that implements the Hadoop FileSystem API. The most common filesystems used
+with GeoMesa FSDS are:
+
+* **HDFS** - Hadoop Distributed File System
+* **S3** - Amazon Simple Storage
+* **GCS** - Google Cloud Storage
+* **WASB** - Windows Azure Blob Store
+* **Local** - Locally Mounted File System (e.g. local disk or NFS)
+
+Choosing a filesystem depends generally on cost and performance requirements. One thing to note is that S3, GCS, and
+WASB are all "cloud-native" storage meaning that they are built into Amazon, Google, and Microsoft Azure cloud
+platforms. These cloud-native filesystems are scaled separately from the compute nodes which generally provides a more
+cost efficient storage solution. In general their price per GigaByte of storage is lower then using HDFS though they
+may be slightly more latent. They also have the ability to persist data after you turn off all your compute nodes.
+
+Any of the filesystems mentioned about are good choices for the FSDS. If you have more questions about making a choice
+contact the `GeoMesa team <http://www.geomesa.org/community/>`__
+
+Partition Schemes
+-----------------
+
+The partition scheme defines how data is stored within the filesystem. The scheme is important because it defines how
+the data is queried. Most OLAP queries in GeoMesa contain a date range and geometric predicate and the partition scheme
+can aide in finding the files satisfy the query. There are two main partition schmes used with GeoMesa FSDS:
```

Comment:
NB:  aide is a noun; you are looking for the verb 'aid'.  http://writingexplained.org/aid-vs-aide-difference